agent/src/discovery_handler_manager/diagrams/dh_device.mmd

```diff
@@ -0,0 +1,14 @@
+sequenceDiagram
+    Discovery Handler ->> DiscoveryHandlerRequest: send discovered devices
+    DiscoveryHandlerRequest ->> DiscoveryHandlerRequest: Updates aggregated list of discovered devices
+    DiscoveryHandlerRequest -) Device Manager: Notifies and updates list of discovered devices
+    DiscoveryHandlerRequest -) Configuration Controller: Requests reconciliation of Configuration linked to Request
+    note over Configuration Controller: The following is Configuration Controller behavior
+    activate Configuration Controller
+    Configuration Controller ->> Configuration Controller: Reconcile Configuration
```

Comment:
What does Reconcile Configuration mean? Is this where we are making sure that the appropriate set of Akri Instances exists?

Reply:
This is just what the configuration controller do, so ensuring there is a request for the configuration and that the appropriate set of Instances exists.
At first I didn't want to put it in that diagram as it is outside of the discovery handler manager, but did it in the end to highlight the `get_request()` and `get_instances()` use.